Movie_MVVM/Movie_MVVM/DescriptionModule/ViewModel/DescriptionViewModel.swift

```diff
@@ -0,0 +1,59 @@
+// DescriptionViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Вью модель экрана описания фильма
+final class DescriptionViewModel: DescriptionViewModelProtocol {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let zeroInt = 0
+    }
+
+    // MARK: - Public Properties
+
+    var updateView: (() -> ())?
```

Comment:
```suggestion
    var updateViewHandler: (() -> ())?
```
и далее по проекту...